src/assets/styles/styles/custom/span-errors.scss

```diff
@@ -0,0 +1,42 @@
+.MuiDrawer-paper {
+  width: 30%;
+}
+
+hr {
```

Comment:
W sumie nie dużo to zmienia, ale myślę, że zamiast stylować `hr` mozna by było użyć https://material-ui.com/components/dividers/
Ale to tak, tylko sugestie na przyszłość

Reply:
Masz racje, już zamieniłem na `Divider`